crates/biome_css_parser/tests/css_test_suite/ok/at_rule/at_rule_container_complex.css

```diff
@@ -0,0 +1,14 @@
+@container name  (width <= 500px) {  }
+@container name  (width <= 500px) and (width >= 600px) or (inline-width >= 300em) {  }
+@container name  (width <= 500px) and (width >= 600px) or (not (inline-width >= 300em)) {  }
+@container name  (width <= 500px) and (width >= 600px) or (not (inline-width >= 300em)) and (width <= 500px) {  }
+@container name not ((width <= 500px) and (width >= 600px)) {  }
+@container name style(not (--a: blue)) {  }
+@container style(--accent-color: blue) {}
+@container name style((--a: blue) and (--b: red)) {  }
+@container name style((--a: blue) and (--b: red) or (--c: green)) {  }
+@container name style((not (--b: red)) or (--c: green)) {  }
+@container name style((not (--b: red)) or (not (--c: green))) {  }
+@container name style((not (--b: red)) or (not (--c: green)) and (background: red)) {  }
+@container name style(not (not (--b: red))) {  }
```

Comment:
SICK!

Reply:
Where do you find all these examples? :D